knip.ts

```diff
@@ -1,5 +1,6 @@
 import type { KnipConfig } from 'knip' with { 'resolution-mode': 'import' };
 
+// eslint-disable-next-line import/no-default-export
```

Comment:
Oh yeah, I fixed this in https://github.com/typescript-eslint/typescript-eslint/pull/9848 too - by just excluding `knip.ts` from `import/no-default-export`... let me merge that now.